mypy.ini

```diff
@@ -9,4 +8,9 @@ show_error_codes = True
 ignore_errors = True
 
 [mypy-evergreen.cli.*]
-ignore_errors = True
+disallow_incomplete_defs = False
+disallow_untyped_defs = False
+disallow_untyped_calls = False
```

Comment:
This will allow the cli entrypoint to be sort-of type checked, should surface errors that wouldn't appear otherwise until the task that actually tries installing the package and running the command.